src/Arcus.Security.Providers.AzureKeyVault/Arcus.Security.Providers.AzureKeyVault.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.4.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.8" />
-    <PackageReference Include="Polly-Signed" Version="5.9.0" />
+    <PackageReference Include="Polly" Version="7.2.1" />
```

Comment:
Just wondering if we should specify the version explicitly, or do something like `7.*` ?

Reply:
NuGet will allow for >7.2.1 but I presume you want this because you are on a lower version?

Reply:
No, I'm not at a lower version.  
So, then it is fine for me I guess.